app/src/test/java/org/simple/clinic/summary/teleconsultation/contactdoctor/ContactDoctorUpdateTest.kt

```diff
@@ -132,4 +132,42 @@ class ContactDoctorUpdateTest {
             hasEffects(SendTeleconsultMessage(patientTeleconsultInfo, MessageTarget.SMS))
         ))
   }
+
+  @Test
+  fun `when whats app button is clicked, then create teleconsult record`() {
+    val medicalOfficerId = UUID.fromString("30831b1c-d92c-48e4-953d-081ae9684c42")
```

Comment:
Can we add `medicalOfficerId` and `doctorPhoneNumber` as global var properties, since we're using them in multiple tests?

Reply:
Hey, we've added them as global properties but haven't removed them from tests? Can you check please

Reply:
we have removed it, please check the commit.

https://github.com/simpledotorg/simple-android/pull/1869/commits/1eff30fba0679b759def9bb9beeb46540a709af8#diff-62eb1732d41c3dc8e17fac31f7171bbdR6